unittests/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ set(unittests
   unittest_ypipe
   unittest_poller
   unittest_mtrie
+  unittest_ip_resolver
+  unittest_ip_resolver_dns
```

Comment:
@simias move unittest_ip_resolver_dns below, like this:

```
IF (NOT WIN32)
    list(APPEND unittests
        unittest_ip_resolver_dns
    )
ENDIF (NOT WIN32)
```

so that building this is skipped with VS which doesn't like the mock